### PR TITLE
{bio}[system/system] MEGAN v6.22.0

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MEGAN/MEGAN-6.22.0-Java-17.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+install_cmd = 'chmod +x %s && ./%s -q -dir %%(installdir)s' % (local_installer, local_installer)
+
+postinstallcmds = ["rm %(installdir)s/uninstall"]
+
+sanity_check_paths = {
+    'files': ['MEGAN'],
+    'dirs': ['jars', 'jars2', 'tools'],
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/m/MEGAN/MEGAN-6.22.0-Java-17.eb:27]: Would it make sense to use one of the tools from the `tools` dir as a sanity check command? From what I saw in my test installation, these are bash scripts that call the same jar file as the `MEGAN` executable itself. And in contrast to `MEGAN`, these tools do seem to have a `--help`/`--version`, so perhaps it's still a good check to find out if the jar file can be called?

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/25ec4ce9d9db5a68fc5456d4bcfd6739 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/7916b15cdcbe03c57566469f9905e875 for a full test report.

**Comment by bedroge** [comment]
Going in, thanks @lexming!

**Pull request merged**